sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormat.scala

```diff
@@ -163,6 +163,12 @@ trait FileFormat {
    * By default all data types are supported.
    */
   def supportDataType(dataType: DataType): Boolean = true
+
+
+  /**
+   * Check whether target schema field name is valid.
```

Comment:
can we follow the doc of the method above?

Reply:
Done